src/components/v1/distribution/distribution.service.ts

```
@@ -0,0 +1,98 @@
+      const rate = Number.parseFloat(usdRate)
+      const totalUsd = amount * rate
+      return totalUsd.toString()
+    } catch (error) {
+      console.warn("Error calculating total USD amount:", error)
+      return "0"
+    }
+  }
+
+  private processMetadata(metadata: Record<string, any>): Record<string, any> {
+    const processedMetadata = { ...metadata }
+    delete (processedMetadata as any).__proto__
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)